commands/build.go

```diff
@@ -832,3 +832,10 @@ func printValue(printer printFunc, version string, format string, res map[string
 	}
 	return printer([]byte(res["result.json"]), os.Stdout)
 }
+
+func getInStream(dockerCli command.Cli) io.ReadCloser {
+	if dockerCli.In() != nil {
```

Comment:
Don't think this is needed, `In()` is always set in cli: 

* https://github.com/docker/buildx/blob/341fb65f6f568696ed5e2585eb747aedaed7ee1a/cmd/buildx/main.go#L54
* https://github.com/docker/cli/blob/5be21394cbbafc5627bb3e8a54014c108a534b52/cli/command/cli.go#L494
* https://github.com/docker/cli/blob/master/cli/command/cli_options.go#L16C1-L26

Reply:
Thanks for a catch! Actualized code.